examples/local_helpers_stream.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import asyncio
+from asyncio import to_thread
+from typing import AsyncIterator
+
+import numpy as np
+import sounddevice as sd
+import torch
+from context import iftk
+
+from iftk.helpers import pyaudio, silero_vad, transformers, whisper, xtts
+
+MODEL_ID = "TinyLlama/TinyLlama-1.1B-Chat-v1.0"
+CHUNK = 512 * 100
+RATE = 16000
+messages = [{"role": "system", "content": "Respond in a few sentences."}]
+
+
+async def main():
+    microphone_stream: AsyncIterator = pyaudio.microphone(
+        rate=RATE, frames_per_buffer=CHUNK
+    )
+    vad_stream: AsyncIterator = silero_vad.silero_vad_stream(
+        microphone_stream, sample_rate=RATE
+    )
+    whisper_stream: AsyncIterator = whisper.whisper_stream(microphone_stream)
+    turn_transcription = ""
+    async for output in whisper_stream:
+        turn_transcription += output
+        turn_transcription += " "
```

Comment:
A common, and often more efficient, way to handle this pattern is to build an array of words and then join them with a space. 
```suggestion
        turn_transcription_parts = []
        turn_transcription_parts.append(output)
        # ... later on
        turn_transcription = " ".join(turn_transcription_parts)
```